### PR TITLE
Use arbitrary namespace for horizon kuttl tests

### PR DESCRIPTION
**Pull request by kajinamit**
... instead of using the static openstack namespace.

### CODE REVIEW

**Comment by kajinamit** [comment]
I've pushed https://github.com/openstack-k8s-operators/horizon-operator/pull/174 to remove the hard-coded namespace (though it seems removing it is not required to run the test in an arbitrary namespace).

File: Makefile [hunk 1]
```diff
@@ -1402,9 +1403,10 @@ glance_kuttl: kuttl_common_prep glance glance_deploy_prep ## runs kuttl tests fo
 
 .PHONY: horizon_kuttl_run
 horizon_kuttl_run: ## runs kuttl tests for the horizon operator, assumes that everything needed for running the test was deployed beforehand.
-	kubectl-kuttl test --config ${HORIZON_KUTTL_CONF} ${HORIZON_KUTTL_DIR}
+	kubectl-kuttl test --config ${HORIZON_KUTTL_CONF} ${HORIZON_KUTTL_DIR} --config ${HORIZON_KUTTL_CONF} --namespace ${NAMESPACE}
 
 .PHONY: horizon_kuttl
+openstack_kuttl: export NAMESPACE = ${HORIZON_KUTTL_NAMESPACE}
```
Comment (abays) [inline Makefile:1409]: `s/openstack_kuttl/horizon_kuttl`

**Review by abays — APPROVED** [review]
/lgtm

**Comment by kajinamit** [comment]
recheck

**Comment by abays** [comment]
recheck

**Comment by kajinamit** [comment]
recheck

**Pull request merged**